packages/api/src/common/constants.ts

```diff
@@ -1 +1 @@
-export const L2_ETH_TOKEN_ADDRESS = "0x000000000000000000000000000000000000800a";
+export const BASE_TOKEN_L2_ADDRESS = "0x000000000000000000000000000000000000800A";
```

Comment:
Any reason to make it upper case?

Reply:
The reason is to match with the checksum address.

It was needed since we use this value a in very different places, and in some of them the address (basically, whenever we use the Repository results) a transformer to checksum address is applied.

Reply:
Why is it renamed? The value is still L2 ETH address.

Reply:
Because it may not represent exactly ETH, but whatever the token is defined when deploying a new L2.

The rename was introduced for a better representation.

Reply:
That particular name was decided here: https://github.com/matter-labs/block-explorer/pull/172#discussion_r1504456855